### ISSUE TITLE
Authority for municipality codes?

### ISSUE THREAD

**Issue by j3h4** [HL7/dk-core-r4#3]
Currently the system for the municipality code system is "http://danmarksadresser.dk/kommunekode".

But after the Greenland municipalities have been added, wouldn't it be more appropriate to use "http://cpr.dk/myndigheder" instead? The CPR Myndighedsregister contains both Danish and Greenland municipalities.

**Comment by jvitrifork** [comment]
I don't believe the CPR Myndighedsregister is the proper authority. I would really like to model this differently with multiple codesystems and multiple concepts in a single valueset bound to multiple valuesets. It would fit the same use case but be more easily understandable to use as it would be easy to filter away eg. danish or greenlandish codes away

**Comment by jvitrifork** [comment]
I believe the BBR (https://ki.bbr.dk/kodelister-bbr/0/1/0/Kommunekode) would be the proper authority under the Indenrigsministeriet for danish kommunekoder

**Comment by j3h4** [comment]
The page [https://cpr.dk/cpr-systemet/bopaelsregistreringen-baseres-paa-dar/] states:

"DAR-loven gælder ikke for Grønland. Bopælsregistreringen i en grønlandsk kommune er således fortsat baseret på CPR's vejregister."

So maybe we should have two code systems, as you mentioned, the Danish with the system "http://danmarksadresser.dk/kommunekode", and the Greenlandish with the system "http://cpr.dk/kommunekoder/greenland" or something similar?

**Comment by jkiddo** [comment]
I think that sounds great ;)

**Comment by jvitrifork** [comment]
The correct DAR address for kommunekode is https://danmarksadresser.dk/adressedata/kodelister/kommunekodeliste/

**Comment by j3h4** [comment]
I have created a new code system for the Greenland municipalities and updated the value set to include both the Danish and the Greenlandish code systems.
